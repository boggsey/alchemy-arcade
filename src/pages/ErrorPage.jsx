import React from 'react';

const ErrorPage = () => {
  return (
    <div>
      This is the error page!
    </div>
  )
}

export default ErrorPage;