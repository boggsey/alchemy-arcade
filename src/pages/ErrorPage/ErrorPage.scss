@import '../../styles/variables.scss';

.cabinet-screen.error-page {
  background: $error-blue !important;
  
  #stars, #stars2, #stars3 {
    display: none;
  }

  .message {
    text-align: center;
    font: 400 24px/36px 'Press Start 2P', Arial;
    color: #fff;
  }
}
