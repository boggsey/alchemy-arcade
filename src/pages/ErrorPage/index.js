import React from 'react';
import { Redirect } from 'react-router-dom';
import './ErrorPage.scss';

const ErrorPage = () => (
  <div className="message">
    {'This isn\'t the component you\'re looking for.'}
    <Redirect to="/error" />
  </div>
);

export default ErrorPage;
