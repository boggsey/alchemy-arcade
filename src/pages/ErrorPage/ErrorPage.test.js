import React from 'react';
import { shallow } from 'enzyme';
import ErrorPage from '.';

describe('ErrorPage', () => {
  it('Hasn\'t changed since the last snapshot', () => {
    const wrapper = shallow(<ErrorPage />);

    expect(wrapper).toMatchSnapshot();
  });
});
