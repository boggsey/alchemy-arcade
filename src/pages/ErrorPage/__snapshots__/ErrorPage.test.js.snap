// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage Hasn't changed since the last snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ErrorPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        "This isn't the component you're looking for.",
        <Redirect
          push={false}
          to="/error"
        />,
      ],
      "className": "message",
    },
    "ref": null,
    "rendered": Array [
      "This isn't the component you're looking for.",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "push": false,
          "to": "/error",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "This isn't the component you're looking for.",
          <Redirect
            push={false}
            to="/error"
          />,
        ],
        "className": "message",
      },
      "ref": null,
      "rendered": Array [
        "This isn't the component you're looking for.",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "push": false,
            "to": "/error",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
